### PR TITLE
feat: Fully implemented IncrementalJavaCompiler

### PR DESCRIPTION
**Pull request by tyron12233**
Compilation of java files is fully working and is being generated on `/mysc/incremental/classes` however it's not yet integrated with Dp.java

### CODE REVIEW

File: app/src/main/java/mod/tyron/compiler/IncrementalJavaCompiler.java [hunk 1]
```diff
@@ -235,8 +352,110 @@ public CompilerOutputStream(StringBuffer buffer) {
...
+            n.a("gson-2.8.0");
+        }
+        if (projectConfig.N.n) {
+            n.a("glide-4.11.0");
+        }
+        if (projectConfig.N.p) {
+            n.a("okhttp-3.9.1");
+        }
+        ExtLibSelected.a(projectConfig.N.x, n);
+    }
```
Comment (JavkhlanK) [inline app/src/main/java/mod/tyron/compiler/IncrementalJavaCompiler.java:460]: These methods seem to be copied from Dp.java. Could you use them instead, as you have a Dp object anyway?

File: app/src/main/java/mod/tyron/compiler/Util.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package mod.tyron.compiler;
+
+import java.io.File;
+
+public class Util {
+
+}
```
Comment (JavkhlanK) [inline app/src/main/java/mod/tyron/compiler/Util.java:7]: What's this class for?

**Review by JavkhlanK — CHANGES_REQUESTED** [review]
There are 2 files that should be changed first. The rest of the files seem good, though.

File: app/src/main/java/mod/tyron/compiler/Util.java (hunk 2 above)
Comment (tyron12233) [reply app/src/main/java/mod/tyron/compiler/Util.java]: Unused class, it will be used in the future ill remove it on this commit

**Review by tyron12233 — COMMENTED** [review]
Removed dp methods

**Review by JavkhlanK — APPROVED** [review]
Seems good now :+1:

**Pull request merged**